### PR TITLE
Better HTML examples

### PR DESCRIPTION
**Pull request by daverau-optimizely**
cc @kwalker3690 @danoc This technically is a breaking change with the classname fix, but i'm calling this a patch for simplicity. 

I'm adding tokens and badges to OUI now and will release a new version shortly with those two pull requests.

### CODE REVIEW

**Comment by daverau-optimizely** [comment]
Here's a preview once I update scribesass and deploy:
screenshot 2016-06-17 13 02 55

File: src/oui/partials/trumps/_type.scss [hunk 1]
```diff
@@ -43,6 +43,29 @@
...
+///   <ul class="badge"><li class="color-admin--background">Admin</li></ul>
+/// @example[html] .color-admin--border
+///   <div class="attention color-admin--border">Some Gated Feature Box</div>
+/// @advanced
+
+.color--admin {
+  color: map-fetch($color, ui, admin) !important;
+}
+
+.color-admin--border {
```
Comment (kwalker3690) [inline src/oui/partials/trumps/_type.scss]: We should be consistent here, either `.color-admin`, `.color-admin--border` or `.color--admin`, `.color--admin--border`

**Comment by kwalker3690** [comment]
Why is it simpler for this to be a patch instead of a breaking change?

**Comment by kwalker3690** [comment]
LGTM, I guess just patching it is ok because its a quick follow up, but technically is a breaking change...but yeah let's keep it to a patch increase

**Comment by daverau-optimizely** [comment]
Updated admin changelog entry to feature, for correctness.

**Pull request merged**